tasks.py

```diff
@@ -210,22 +210,27 @@ def mv_usage_stats_json(ctx):
 @invoke.task(
     aliases=["test"],
     help={
+        "unit": "Runs tests marked with the 'unit' marker. Default behavior.",
         "integration": "Runs integration tests and exclude unit-tests. By default only unit tests are run.",
+        "ignore-markers": "Don't exclude any test by not passing any markers to pytest.",
```

Comment:
Right now `-m 'unit'` is the default marker that gets passed to `pytest`, but should I change this so that `--unit/-u` or `--integration` must be explicitly passed?

Then we could get rid of the `--ignore-markers` flag.